certificate-policy.md

```diff
@@ -1089,6 +1095,52 @@ The business continuity plan MUST include:
 14. The distance of recovery facilities to the CA's main site; and
 15. Procedures for securing its facility to the extent possible during the period of time following a disaster and prior to restoring a secure environment either at the original or a remote site.
 
+The FPKIPA shall be notified by the CAs operating under this policy of any security incident. A security incident or incident is defined as a violation or imminent threat of violation of the NPE CP, CPS, subscriber agreements, MOA, or any other document that governs the operations of the CA. A security incident may include but is not limited to the following:
+- Suspected or detected compromise of Certificate Systems
+- Suspected or detected compromise of a certificate status server (CSS) if:
+  - The CSS certificate has a lifetime of more than 72 hours and
+  - The CSS certificate cannot be revoked (e.g., an OCSP responder certificate with the id-pkix-ocsp-nocheck extension)
+- Physical or electronic penetration of the Certificate Systems
+- Successful denial of service attacks on the Certificate System components
+- Any incident preventing the CA from issuing a CRL within 48 hours of the issuance of the previous CRL
+- Suspected or detected issuance of fraudulent certificates used for unethical purposes such as but not limited to promoting malware or illegal software.
```

Comment:
> Suspected or detected issuance of fraudulent certificates used for unethical purposes such as but not limited to promoting malware or illegal software.

I'm concerned about this one. For one, I think we should avoid the use of `unethical`, since that is a more subjective line than I think we intend here. Are we really expecting CAs to judge the _ethicality_ of the way their certificates are used by other agencies?

In addition, even if the certificate were being used for something outright _illegal_ or if the government service in question was obviously hacked, I don't think that would be a security incident of the CA itself.

In other words:

* If someone pops state.gov and hosts malware on it, and state.gov is using a certificate from our infrastructure, that is definitely a security incident for State -- but **not** an incident for the CA or the root.
* If someone pops State's DNS provider, sets up a new domain at softwareupdates.state.gov to try to spearphish State employees, and gets a DV certificate from an issuing CA in this hierarchy by proving domain control -- that is most definitely a security incident for State -- but **not** an incident for the CA or the root.

In neither case above did the issuing CA or the root do anything "wrong", and it shouldn't trigger the same path as an actual security incident of the issuing CA's infrastructure would.